app/gui/view/debug_scene/component-group/src/lib.rs

```diff
@@ -343,16 +369,63 @@ fn init(app: &Application) {
     }
 
 
+    // === Selection box ===
+
+    let selection = component_group::selection_box::View::new(&app.logger);
+    selection.size.set(Vector2(150.0, list_view::entry::HEIGHT));
+    selection_mask.add_exclusive(&selection);
+    app.display.add_child(&selection);
+
+    let selection_animation = Animation::<Vector2>::new(&network);
+    selection_animation.set_spring.emit(SELECTION_ANIMATION_SPRING_FORCE);
+    /// This is an example code to position the selection box on scene.
+    /// `multiview.selection_position_target` returns a group-local position, so we transform it
+    /// to global one and then restrict the y coordinate so that the selection box would not go
+    /// below the scroll area bottom.
```

Comment:
It's really hard to read these sentences. Please make them shorter and run them trough a grammar checking tool. Gramarly is cool. If you need pro license, Ola will buy you one.

Reply:
Done.